.gitpod.yml

```
@@ -1,15 +1,12 @@
-image: gitpod/workspace-full:2022-08-04-13-40-17
+image: 
+  file: gitpod.Dockerfile
 workspaceLocation: observability/observability.code-workspace
 tasks:
-  - init: |
-      # TODO(arthursens): We should update the workspace image so we dont need the steps below
-      sudo curl -Lo /usr/bin/yq https://github.com/mikefarah/yq/releases/download/v4.27.2/yq_linux_amd64 && sudo chmod +x /usr/bin/yq
-      sudo apt update ; sudo apt-get install shellcheck apt-transport-https ca-certificates -y ; sudo update-ca-certificates
+  - name: Install pre-commit hooks & generate manifests
+    command: |
```

```suggestion
    init: |
```

Sorry, I've also missed this. I think prebuilds are still very important to have... although they are a bit unstable at the moment it is one of the core functionalities of Gitpod. We should dogfood our own product and provide feedback when things don't work as needed 😅.

Now that I finally joined the Gitpod team I can see [why our prebuilds](https://gitpod.io/t/gitpod/observability/prebuilds) were taking so much time, and looks like our own setup is broken 😬

---

Yeah, I forgot to comment on that.

I would suggest tackling this in another issue so at least the WS works until we fix the prebuilds